app/models/host.rb

```diff
@@ -1701,11 +1691,15 @@ def get_pct_cpu_metric_from_child_vm_performances(metric, capture_interval, time
   # Host Discovery Types and Platforms
 
   def self.host_discovery_types
-    HOST_DISCOVERY_TYPES.values
+    # TODO: This feature has been removed, once the UI no longer calls this
+    # method we can delete it
+    []
```

Comment:
Should we emit a deprecation warning? (See Vmdb::Deprecation)

Reply:
Done